theme_5/scss/_modules.scss

```diff
@@ -0,0 +1,217 @@
+@import "burger";
+
+.burger-icon {
+	@include burger(25px, 3px, 5px, $white-solid);
+}
+
+.menu-button {
+	background-color: $cerise-red;
+	border: 1px solid $cerise-red;
+	display: inline-block;
+	padding: .5em;
+	&.is-active {
+		.burger-icon {
+			@include burger-to-cross;
+		}
+	}
+}
+
+.badge-conf{
+	color: $white-solid;
+	line-height: 50px;
+	position: absolute;
+	right: 50px;
+}
+
+.logo {
+  background: url(../assets/logos_png/logo1_fiorella_mobil.png) no-repeat center;
+  height: 50px;
+}
+
+// Sprites
+.cart-icon {
+	background: url(../assets/Sprite.png) no-repeat -171px -383px;
+	height: 19px;
+	line-height: 50px;
+	position: absolute;
+	top: 15px;
+	width: 20px;
+
+}
+
+.bus-icon {
+	background: url(../assets/Sprite.png) no-repeat -224px -48px;
+	width: 68px;
+	height: 52px;
+	transform: scale(.7);
+}
+// Input
+.container-text {
+	padding: 7px 7px;
+	margin: 0;
+	.text-box {
+		background: url(../assets/Sprite.png) no-repeat 230px -50px;
+		box-shadow: none;
```

Comment:
Properties should be ordered background, border, border-bottom, box-shadow, display, font-family, font-size, font-style, margin, outline, padding, text-transform